src/Raven.Client/ServerWide/CompactSettings.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents
+{
+    public class CompactSettings
+    {
+        public string DatabaseName { get; set; }
+
+        public bool Documents { get; set; }
+
+        public string[] Indexes { get; set; }
```

Comment:
How do I compact _all_ indexes?

Reply:
Send this json for example:
{
	"DatabaseName" : "Northwind",
	"Documents" : true,
	"Indexes" : ["Orders/ByCompany", "Orders/Totals" , "Product/Sales"]
}